src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -76,12 +76,20 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     void MoveSizeStart(HWND window, HMONITOR monitor, POINT const& ptScreen) noexcept
     {
         std::unique_lock writeLock(m_lock);
+        if (m_settings->GetSettings()->multiMonitorMode)
+        {
+            monitor = NULL;

```

Comment:
This looks wrong.. Setting function argument to NULL and pass it further..

Reply:
`NULL` is used to signal that the function is being called in multi-monitor mode, so that I don't have to read the settings every time and also so I don't pass around another `bool` parameter. This same pattern appears in a few other places with a `NULL` `HMONITOR`. I'll consider what can be done to make reading the code easier.

Reply:
`NULL is used to signal that the function is being called in multi-monitor mode`
Yes, that is what is wrong. Check if code could be reorganized to fit this purpose and/or introduce enum to signalize multi-monitor mode

Reply:
Actually, since `HMONITOR` is being used as a key for the map in `MonitorWorkAreaHandler`, this would require changing it as well, further complicating the code, so it's much more than just parameters to some functions. I think that the best thing we can do now is to just add comments where `NULL` handles are used to explain that this means that the handle refers to no specific monitor - meaning that the specific class/function is running across all monitors.